src/Client.php

```diff
@@ -308,10 +308,16 @@ public function getAccessToken($code = '')
      */
     public static function responseToArray($response)
     {
-        return \GuzzleHttp\json_decode(
-            $response->getBody()->getContents(),
-            true
-        );
+        if ($contents = $response->getBody()->getContents()) {
+            return \GuzzleHttp\json_decode(
+                $contents,
+                true
+            );
+        }
+        elseif ($contents = $response->getHeaders()) {
```

Comment:
I think we can omit `else` here and use just `if` statement. Because inside the first `if`-statement we return result and stop performing this method.

Reply:
We still have the same problem: we need to parse the `$response->getHeaders` in case the body is `$response->getBody()->getContents()`is null, so we have to make another check.

We can just return the `$response->getHeaders()` with no `else`, but I cannot be sure if it would break another api call.

Reply:
It seems we have some misunderstanding. I said about something like that:

```php
if ($contents = $response->getBody()->getContents()) {
    return ....;
}

if ($contents = $response->getHeaders()) {
     return ....;
}

return [];
```

Reply:
Ok, I get it, you're right. I'll make another pull request with your suggestion.

Thx!